src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -282,7 +319,15 @@ internal int SerializeLogRecord(LogRecord logRecord)
         }
 
         // Part A - core envelope
-        cursor = AddPartAField(buffer, cursor, Schema.V40.PartA.Name, eventName);
+        if (sanitizedEventName != default)
```

Comment:
nit: I think `if (sanitizedEventName.Length > 0)` would be slightly faster and good enough for the algorithm. The [operator implementation](https://github.com/dotnet/runtime/blob/110b4cfaa627deed4b06b864b06128267d0a64c2/src/libraries/System.Private.CoreLib/src/System/Span.cs#L359) first has to copy both values, makes the call, then checks the lengths, and then does some kind of fancy reference check. Probably very minor difference in speed but hey 😄

Reply:
Oh wow :star_struck:!! Thank you so much for the code pointer.